src/compiler/diagnosticMessages.json

```diff
@@ -5665,7 +5665,15 @@
         "category": "Message",
         "code": 95116
     },
-    
+    "Wrap elements in JSX fragment": {
+        "category": "Message",
+        "code": 95117
+    },
+    "Wrap all elements in JSX fragment": {

```

Comment:
Maybe “Wrap all **unparented** elements in JSX fragment**s**” would be better? Paging @DanielRosenwasser for opinions on the words